.circleci/config.yml

```diff
@@ -0,0 +1,114 @@
+version: 2.1
+orbs:
+  node: circleci/node@5.0.2
+jobs:
+  build:
+    executor: node/default
+    steps:
+      - checkout
+      - node/install-packages:
+          pkg-manager: npm
+      - run:
+          command: sudo npm install -g ts-node
+          name: Install ts-node
+      - run:
+          command: ts-node -v
+          name: Check ts-node version to make sure it installed
+      - persist_to_workspace:
+          root: .
+          paths:
+            - '*'
+
+  test:
+    executor: node/default
+    steps:
+      - attach_workspace:
+          at: .
+      - checkout
+      - run:
+          command: npm run test
+          name: run tests
+
+  format:
+    executor: node/default
+    steps:
+      - attach_workspace:
+          at: .
+      - checkout
+      - run:
+          name: format
+          command: npm run format
+
+  lint:
+    executor: node/default
+    steps:
+      - attach_workspace:
+          at: .
+      - checkout
+      - run:
+          name: lint
+          command: npm run lint
+
+  integration:
+    executor: node/default
+    steps:
+      - attach_workspace:
+          at: .
+      - checkout
+      - run:
+          name: testnet
+          command: |
+            export AUTH_HEADER=$AUTH_HEADER_TESTNET
+            export WALLET_SECRET_KEY=$PRIVATE_KEY
+            export WALLET_PUBLIC_KEY=$PUBLIC_KEY
+            export API_ENV="testnet"
+            export RUN_LIFECYCLE="false"
+            npm start
+          no_output_timeout: 30m
+      - run:
+          name: mainnet
+          command: |
+            export AUTH_HEADER=$AUTH_HEADER_MAINNET
+            export WALLET_SECRET_KEY=$PRIVATE_KEY
+            export WALLET_PUBLIC_KEY=$PUBLIC_KEY
```

Comment:
do we expect to deploy to mainnet before merging the SDK?

Reply:
we expect integration tests to pass against mainnet, no?

Reply:
Not necessarily. We might have breaking changes from testnet to mainnet, but we should at least know if they pass against mainnet or testnet or not. You should probably separate mainnet from testnet runs, then we can make testnet mandatory, but mainnet optional.